packages/proxy/lib/network-proxy.ts

```diff
@@ -41,4 +41,8 @@ export class NetworkProxy {
   reset () {
     this.http.reset()
```

Comment:
do we need to unset the protocol manager when the http middleware is reset to prevent bleedover? Though I am not to sure the cases where this happens in run mode

Reply:
The protocol manager should be good for the life of the run. I don't think there's a reason to reset it.

Reply:
OK if not we might have to add it, but usually this is only a thing for open mode afaik